openssh/files/default/ssh_known_hosts

```diff
@@ -81,26 +81,19 @@
 {%- set host_names = salt['mine.get'](target, hostname_function, tgt_type=tgt_type) -%}
 
 {#- Salt SSH (if any) #}
-{%- for minion_id, minion_host_keys in salt['pillar.get'](
-  'openssh:known_hosts:salt_ssh:public_ssh_host_keys',
-  {}
-).items() -%}
+{%- for minion_id, minion_host_keys in known_hosts | traverse('salt_ssh:public_ssh_host_keys', {}) | dictsort -%}
```

Comment:
It would be interesting to hear the rationale about using `| dictsort` here.  Is it necessary?  Need to be aware that there are issues with using `dictsort` on Python 3 as well:

* https://github.com/saltstack-formulas/bind-formula/issues/146

Haven't dug down to see if any of that applies here but couldn't we stick to using `.items()`?

Reply:
Hello, I finished by using `| dictsort` because `known_hosts | traverse('salt_ssh:public_ssh_host_keys', {}).items()` is not working.

I'm confused that there is no jinja filter `| items` to apply on `dict` and so using `.items()` requires to define an intermediate variable.

At least, `dictsort` did the job but I will pass by intermediate variables to avoid problems with mixed keys as describe is the [bind-formula issue](https://github.com/saltstack-formulas/bind-formula/issues/146).

Reply:
I used an intermediate variable.